commands/cdns.go

```diff
@@ -29,34 +29,68 @@ func CDN() *Command {
 	cmd := &Command{
 		Command: &cobra.Command{
 			Use:   "cdn",
-			Short: "cdn commands",
-			Long:  "cdn is used to access cdn commands",
+			Short: "Provides access to commands that manage CDNs.",
```

Comment:
```suggestion
			Short: "Create, access, and manage CDN endpoints.",
```

From a UI perspective for these top level short descriptions, it might be nice to be a bit more action orientated and focus on what the task the user wants to acomplish. I'm picking on `cdn` here, but it could be any command. Thoughts @johndmulhausen?

Reply:
My thinking here was that for the top-level "namespaces" that don't actually do anything on their own it created a sort of double-jeopardy to say that this command "creates... CDN endpoints" when it's really just a bucket of commands and `doctl compute cdn create` would actually do that. 

Having one place that promises x functionality seemed "cleanest" to me. Just my $0.02, happy to shift if that's not making sense to anyone but me